packages/node_modules/@webex/webex-core/src/lib/services/service-catalogs.js

```diff
@@ -41,6 +54,57 @@ export default class ServiceCatalogs {
     this.hosts = [];
   }
 
+  /**
+   * An active, local, and priority mapped record of the current
+   * {@link ServiceCatalog#hosts}.
+   *
+   * @public
+   * @memberof ServiceCatalog
+   * @type {Record<string, string>}
+   */
```

Comment:
Does this need a `@returns`?

Reply:
Getter methods act as properties, so they simply need a `@type`